src/FSharpPlus/Control/Functor.fs

```diff
@@ -62,13 +62,13 @@ type Iterate =
 type Map =
     inherit Default1
 
-#if (!FABLE_COMPILER || FABLE_COMPILER_3) && !FABLE_COMPILER_4
+#if !FABLE_COMPILER
 
     static member Map ((x: Lazy<_>             , f: 'T->'U), _mthd: Map) = Lazy.map f x
     #if !FABLE_COMPILER
     static member Map ((x: Task<'T>            , f: 'T->'U), _mthd: Map) = Task.map f x : Task<'U>
     #endif
-    #if !NET45 && !NETSTANDARD2_0 && !FABLE_COMPILER
+    #if !FABLE_COMPILER
```

Comment:
Is this needed since it is in a block that is already excluded from Fable?

Reply:
We can try removing it, once everything is green I guess.